_data/sites.json

```diff
@@ -1520,6 +1520,17 @@
         ]
     },
 
+    {
+        "name": "Bol.com",
+        "url": "https://www.bol.com/nl/rnwy/account/privacy/delete",
+        "difficulty": "hard",
+        "notes": "Need to request deletion on a page within the account settings. Confirm after via link in email, which often doesn't work, because you need to be logged in to the account for the confirmation link to be working. This is not clear.",
```

Comment:
```suggestion
        "notes": "Need to request deletion on a page within the account settings. A link to confirm deletion will be sent to your e-mail after a few days. Confirm deletion in the link, though it is unreliable because you need to be logged in to the account for the confirmation link to be working.",
```

Reply:
This version LGTM also